.github/get_results.php

```diff
@@ -77,20 +99,47 @@ function getTestResultFromFile($file)
 
     return [
         'uuid' => uniqid(),
-        'title' => 'Upgrade to ' . $data[1],
-        'context' => '{"value": "Upgrade to ' . $data[1] . '"}',
+        'title' => 'Upgrade from ' . $data[0] . ' to ' . $data[1],
+        'context' => '{"value": "Upgrade from ' . $data[0] . ' to ' . $data[1] . '"}',
         'skipped' => [],
         'pending' => [],
         'duration' => $duration,
         'state' => $state,
         'err' => $error,
         'date_start' => $dateStart,
         'date_end' => $dateEnd,
-        'branch' => $data[0]
+        'branch' => $data[1]
     ];
 }
 
-function getDateTimeFromString($datetime)
+function getDateTimeFromString(string $datetime): DateTime
 {
     return DateTime::createFromFormat('Y-m-d\TH:i:s\Z', $datetime, new DateTimeZone('UTC'));
 }
+
+function getGlobalResults(array $results, string $branch): array
+{
+    $globalResults = [
+        'title' => 'Upgrade to branch ' . $branch,
+        'date_start' => null,
+        'date_end' => null,
+        'duration' => 0,
+        'passes' => 0,
+        'failures' => 0,
+    ];
+
+    foreach ($results as $result) {
+        if (null === $globalResults['date_start'] || $result['date_start'] < $globalResults['date_start']) {
```

Comment:
Are you sure comparing date like that is a good idea?
Is date_start an int? If not, it should be compare with a DateTime object, wdyt?

Reply:
`DateTime` objects can be compared using comparison operator (https://www.php.net/manual/en/datetime.diff.php#example-2517)

Reply:
Ohhhh these are datetime object :smiley_cat: